base/ltxdoc.dtx

```diff
@@ -300,10 +300,17 @@
 % \begin{macro}{\cs}
 % \changes{v2.0d}{1994/03/17}{Add \cs{cs}}
 % \changes{v2.1h}{2022/06/22}{Update \cs{cmd} definition to work with new \cs{cs}}
+% \changes{v2.1k}{2025/11/18}{Handle \cs{cmd} in bookmarks, gh/877}
 %    \begin{macrocode}
 %\DeclareRobustCommand\cs[1]...                       % defined later
 %\def\cmd#1{\cs{\expandafter\cmd@to@cs\string#1}}     % can't use with new \cs 
 \def\cmd#1{\texttt{\char`\\\expandafter\cmd@to@cs\string#1}}
+%    \end{macrocode}
+% hyperref allow to fill the pdfstringdef-hook before the package is loaded:
+%    \begin{macrocode}
+\providecommand\pdfstringdefPreHook{}
+\g@addto@macro\pdfstringdefPreHook{%  
+  \def\cmd#1{\textbackslash\expandafter\@gobble\string#1}}%
 \def\cmd@to@cs#1#2{\char\number`#2\relax}
```

Comment:
looks all okay, but wouldn't it be better to simply define cmd always in that way?

Reply:
öh sorry I don't understand. The issue says that \textbackslash should not be used for typesetting and the code here is literally your suggestion from the issue?